db/migrate/20240522034351_add_status_to_pet_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToPetApplications < ActiveRecord::Migration[7.1]
+  def change
+    add_column :pet_applications, :status, :string, default: :Pending, null: false
```

Comment:
Adding the status column was helpful for us to be able to retrieve it as an attribute later on!